frontend/deploy-smoke.js

```diff
@@ -30,6 +30,14 @@ driver
     return value;
   })
   .then((value) => {
-    if (value.includes("success")) process.exit(0);
+    if (value.includes("success")) {
+      console.log(`Smoke test returned success status for ${appUrl}`);
+      process.exit(0);
+    }
+    if (value.includes("failure")) {
+      console.log(`Smoke test returned failure status for ${appUrl}`);
+      process.exit(1);
+    }
+    console.log("Smoke test encountered unknown failure.");
     process.exit(1);
```

Comment:
Do we need to add an early return in the it blocks for this not to always run? Don't know the API well enough to know for sure

Reply:
If we used `process.exit()` then this will end the function immediately, but it [looks like the recommended practice](https://stackoverflow.com/a/37592669) is to avoid that if possible and instead just assign the `process.exitCode` so I updated this to use that instead